bitnami/logstash/values.yaml

```diff
@@ -291,6 +291,12 @@ service:
   ## Load Balancer sources
   ## https://kubernetes.io/docs/tasks/access-application-cluster/configure-cloud-provider-firewall/#restrict-access-for-loadbalancer-service
   ##
+
+  ## Enable client source IP preservation
+  ## ref http://kubernetes.io/docs/tasks/access-application-cluster/create-external-load-balancer/#preserving-the-client-source-ip
+  ##
+  externalTrafficPolicy: Cluster
```

Comment:
Additionally to the recommendation above, this should set to `""` by default.

Reply:
Closing this request according to https://github.com/bitnami/charts/pull/5619#discussion_r583637858